node/config/types.go

```diff
@@ -65,6 +65,15 @@ type StorageMiner struct {
 	HarmonyDB HarmonyDB
 }
 
+type LotusProviderConfig struct {
+	EnableWindowPost  bool
+	EnableWinningPost bool
```

Comment:
Maybe put this in a non-top-level ProviderSubsystemsConfig struct?

Reply:
Then what would we munge into the layer's name? (today myLayer.toml becomes myLayer)